### PR TITLE
✨ Dragndrop feature to move a file into a folder (#138)

### LINKED ISSUE #138: ✨ Drag and drop to move files between folders
It might be more intuitive to just drag and drop a file to move it.
Section and move it after validation with the modal could be quiet.

Feature: Drag and drop

  Background :
    Given a directory
    Given a User

  Scenario : User move a file in a folder from the same directory
    Given : Actual directory
    When : User hold the click on the file that he wants to move
    Then : File follow the moving cursor
    Then : User move his cursor on the folder in which he wants to move the file
    When : Cursor his over the folder
    Then : Folder highlight
    When : User opt eh file over the folder
    Then : File is move to the folder
    Then : A little info message appear to confirm the action

https://github.com/linagora/TDrive/assets/52284320/af9ea8e5-0172-4a8a-9d18-2b31c4858bdb

### PR DESCRIPTION
**Pull request by Greemty**
This feature is implemented in its simplest version.

Resume of the feature :
You are able to drag a unique file (not folders) and drop it above a folder. This dragged file will be moved in the folder. A little info message is displayed to confirm the action.

Tools:
- https://dndkit.com/

To do:
- new language variable has been added for the info message. Yet just translated in French and English.
- Add highlights for over folder; need to create a HandleDragOver function...

Possible improvements :
- moving multiple files at once
- moving folder into folder (need of recursion for the folder content)

Demo Video

https://github.com/linagora/TDrive/assets/52284320/d9adc3c3-c503-4c06-adce-4cf714af146d

### CODE REVIEW

**Comment by chibenwa** [comment]
Could you record a little video and upload it here?

It makes it easier to evaluate the resulting UX and is pretty quick to record.

File: tdrive/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
```
Comment (chibenwa) [inline tdrive/package-lock.json:1]: Does it makes sense to commit this file?

**Comment by shepilov** [comment]
@Greemty could you please highlight the place where you are dropping the file and remove all the elements except the file name from the item that you are dragging
image

**Comment by Greemty** [comment]
Now the code for the modal has been simplified. Furthermore, now folder are highlighted when dragging a file over and dragged elements are just the file name.
A video example :

https://github.com/linagora/TDrive/assets/52284320/3a18bf5e-121d-4b7b-9360-6005bf8f6fab

In order to work you must run `yarn add @dnd-kit/core`

**Pull request merged**